.github/CODEOWNERS

```
@@ -0,0 +1,17 @@
+
+# Libraries
+thrust/ @nvidia/cccl-thrust-codeowners
+cub/ @nvidia/cccl-cub-codeowners
+libcudacxx/ @nvidia/cccl-libcudacxx-codeowners
+
+# Infrastructure
+.github/ @nvidia/cccl-infra-codeowners
+ci/ @nvidia/cccl-infra-codeowners
+.devcontainer/ @nvidia/cccl-infra-codeowners
+
+# cmake
+**/CMakeLists.txt @nvidia/cccl-cmake-codeowners
+**/cmake/ @nvidia/cccl-cmake-codeowners
+
+# general codeowners for all files
+* @cccl-codeowners
```

Do we need a newline here?
```suggestion
* @cccl-codeowners

```